contributing/code/experimental.rst

```diff
@@ -0,0 +1,23 @@
+Experimental Features
+=====================
+
+All Symfony features benefits from our :doc:`Backward Compatibility Promise
+</contributing/code/bc>` to give developers the confidence to upgrade to new
+versions safely and more often.
+
+But sometimes, a new feature is controversial. Or finding a good API is not
+easy. In such cases, we want to be able to gather feedback from real-world
+usage, adapt the API, or remove it altogether. Doing so is not possible with a
+no BC-break approach.
+
+To avoid being bound to our backward compatibility promise, such features can
+be marked as **experimental**; classes and methods parts of an experimental
+feature must be marked with the ``@experimental`` tag.
```

Comment:
`...; classes and methods parts of an experimental feature must be marked ...` -> `... and their classes and methods must be marked ...` ?